src/results.cpp

```diff
@@ -336,25 +336,25 @@ util::Optional<Mixed> Results::aggregate(size_t column, bool return_none_for_emp
         throw OutOfBoundsIndexException{column, m_table->get_column_count()};
 
     auto do_agg = [&](auto const& getter) -> util::Optional<Mixed> {
+        util::Optional<Mixed> results;
```

Comment:
It doesn't seem useful to introduce this local when we can just return the values directly.